web/src/components/contactForm.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+
+export default function ContactForm() {
+    const [formData, setFormData] = useState(null);
+    const [isSubmitFormSuccessful, setIsSubmitFormSuccessful] = useState(false);
+
+    const encode = (data) => {
+        return Object.keys(data)
+            .map(
+                (key) =>
+                    `${encodeURIComponent(key)}=${encodeURIComponent(
+                        data[key],
+                    )}`,
+            )
+            .join('&');
+    };
+
+    const handleChange = (event) => {
+        setFormData({ ...formData, [event.target.name]: event.target.value });
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const form = e.target;
+        fetch('/', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+            body: encode({
+                'form-name': form.getAttribute('name'),
+                ...formData,
+            }),
+        })
+            .then(() => setIsSubmitFormSuccessful(true))
+            .catch((error) => error);
+    };
+
+    return (
+        <div className="bg-brand-1 text-white w-80 md:w-[30rem] p-8 rounded-bl-[2.5rem]">
```

Comment:
I don't think this element should align with the center of the page. It should probably be `justify-self: flex-end'